src/freenas/usr/lib/netdata/python.d/truenas_disk_temp.chart.py

```diff
@@ -0,0 +1,52 @@
+from bases.FrameworkServices.SimpleService import SimpleService
+
+from middlewared.utils.disks import get_disks_with_identifiers
```

Comment:
You do not need the `get_disks_with_identifiers`. The object returned from `iterate_disks` has an `identifier` property that can be called which will give the information that you need.